wagtail/wagtailusers/views/users.py

```diff
@@ -126,3 +126,16 @@ def edit(request, user_id):
         'user': user,
         'form': form,
     })
+
+
+@permission_required(delete_user_perm)
+def delete(request, user_id):
+    user = get_object_or_404(User, id=user_id)
+    if request.POST:
```

Comment:
This should be changed to ``if request.method == 'POST':``

This is so the view works, even if there's not ``POST`` data.